InterviewFlashCardsTests/InterviewFlashCardsTests.m

```diff
@@ -22,7 +13,6 @@ @implementation InterviewFlashCardsTests
 
 - (void)setUp {
     [super setUp];
-    self.qman = [IFCQueryManager new];
```

Comment:
Ah, my bad. I wanted to hide the error message that was popping up so I guess I deleted it